src/Sprite.ts

```diff
@@ -552,33 +664,36 @@ export class Sprite extends SpriteBase {
     this._penDown = penDown;
   }
 
-  get penColor() {
+  public get penColor(): Color {
     return this._penColor;
   }
 
-  set penColor(color) {
+  public set penColor(color: unknown) {
     if (color instanceof Color) {
       this._penColor = color;
     } else {
       console.error(
-        `${color} is not a valid penColor. Try using the Color class!`
+        `${String(color)} is not a valid penColor. Try using the Color class!`
       );
     }
   }
 
-  stamp() {
+  public stamp(): void {
     this._project.renderer.stamp(this);
   }
 
-  touching(target, fast = false) {
+  public touching(
+    target: "mouse" | "edge" | Sprite | Stage,
```

Comment:
Out of scope for this PR (since it's sb-edit changes) but I think this is one of the only places we're still using "keywords" like this without an enum.

Reply:
Branched into #169